shopinvader_cart_reminder/models/sale_order.py

```diff
@@ -0,0 +1,77 @@
+# -*- coding: utf-8 -*-
+# Copyright 2019 ACSONE SA/NV (<http://acsone.eu>)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from datetime import timedelta
+
+from odoo import api, fields, models
+
+
+class SaleOrder(models.Model):
+    _inherit = "sale.order"
+
+    last_reminder = fields.Datetime(
+        help="Date/hour of last  (automatic) reminder", default=False
+    )
+
+    @api.model
+    def _get_sale_order_for_reminder(self, backend, reminder_date):
+        """
+        Get sale orders for the reminder
+        :param backend: shopinvader.backend
+        :param reminder_date: str
+        :return: sale.order recordset
+        """
+        anonymous_user = backend.mapped("anonymous_partner_id")
```

Comment:
would name it anonymous_partner. It is confusing to have a res_partner variable called user...